hassio/src/dashboard/hassio-update.ts

```diff
@@ -89,28 +99,38 @@ export class HassioUpdate extends LitElement {
     curVersion: string,
     lastVersion: string,
     apiPath: string,
-    releaseNotesUrl: string
+    releaseNotesUrl: string,
+    icon?: string
   ): TemplateResult {
     if (lastVersion === curVersion) {
       return html``;
     }
     return html`
-      <paper-card heading="${name} update available! 🎉">
+      <paper-card>
         <div class="card-content">
-          ${name} ${lastVersion} is available and you are currently running
-          ${name} ${curVersion}.
+          ${icon
+            ? html`
+                <div class="icon">
+                  <iron-icon .icon="${icon}" />
+                </div>
+              `
+            : ""}
+          <div class="update-heading">${name} ${lastVersion}</div>
```

Comment:
We should do this in the card header, with the default styles.

Reply:
That's essentially how it is rendered now, with a few minor tweaks to what text goes where. I tried that at first but chose not to for a few reasons:

* The default heading styles for the `<paper-card>` seem too large with too much spacing between the heading and the content compared to the tighter design of the mockup, which I prefer
* The heading for the updates section is "Update(s) Available" and each card title should visually indicate that it is a sub-heading of that section for a clearer visual information hierarchy instead of competing with it by having a very similar font size
* Using the card heading would require additional styles applied to place the icon at the top right corner (e.g. right padding for the heading, then absolutely position icon within the card), so we still end up customizing the heading styles if we want to keep the icon in that location
* The default card heading provides no semantic value; it's just a `<div>` without an ARIA heading role/level applied, so putting the heading in the content element doesn't take away any semantic value that I can see

Here is how this looks using the default heading (the icon would still need adjustment/removal):
<img width="718" alt="Screen Shot 2019-10-18 at 11 09 28 AM" src="https://user-images.githubusercontent.com/1522068/67117571-e07aa300-f197-11e9-9ee3-8cf4c78bb1d2.png">

Thoughts?

Reply:
ok, let's keep it how you originally made it, and just make a header bit less bold.

Reply:
This is with `font-weight: 500`
![Screen Shot 2019-10-22 at 1 44 25 PM](https://user-images.githubusercontent.com/1522068/67331207-50f42d80-f4d2-11e9-9781-a35366e4c0e2.png)